kratos/utilities/variable_utils.cpp

```diff
@@ -174,35 +174,27 @@ bool VariableUtils::CheckVariableKeys()
 /***********************************************************************************/
 /***********************************************************************************/
 
-void VariableUtils::UpdateCurrentToInitialConfiguration(const ModelPart::NodesContainerType& rNodes) {
+void VariableUtils::UpdateCurrentToInitialConfiguration(const ModelPart::NodesContainerType& rNodes)
+{
     KRATOS_TRY;
 
-    const int num_nodes = static_cast<int>(rNodes.size());
-    const auto it_node_begin = rNodes.begin();
-
-    #pragma omp parallel for
-    for (int i=0; i<num_nodes; i++) {
-        const auto it_node  = it_node_begin + i;
-        noalias(it_node->Coordinates()) = it_node->GetInitialPosition();
-    }
+    BlockPartition<const ModelPart::NodesContainerType, ModelPart::NodesContainerType::const_iterator>(rNodes).for_each([](Node<3>& rNode){
```

Comment:
I think that the changes in #8251 make possible to use the short syntax. I'd wait for it to be merged.

Reply:
Done